resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -27,16 +27,15 @@ lambda.create.env_vars.label=Environment Variables:
 lambda.create.iam.role.label=IAM Role:
 lambda.create.enable.xray.label=Enable AWS X-Ray
 lambda.create.source_bucket.label=Source Bucket:
-lambda.function.name.tooltip=The function's name, which can contain only the letters A through Z, the letters a through z, the numbers 0 through 9, the hyphen character (-), and the underscore character (_). This name must be less than 64 characters in length.
-lambda.function.description.tooltip=Any meaningful description about the function.
-lambda.function.handler.tooltip=The identifier of the corresponding function handler for Java or the function handler for Python.
-lambda.function.runtime.tooltip=The identifier of the runtime for Lambda to use.
-lambda.function.timeout.tooltip=The amount of time that Lambda allows a function to run before stopping it. Specify an amount of up to 900 seconds (15 minutes).
-lambda.function.memory.tooltip=The amount of memory available to the function during its execution. Specify an amount between 128 MB and 3,008 MB in 64 MB increments.
-lambda.function.env_vars.tooltip=Any environment variables for the function to use, specified as key/value pairs. To add, change, or delete environment variables, choose the folder icon, and then follow the on-screen instructions.
-lambda.function.iamRole.tooltip=Choose an available Lambda execution role in the connected AWS account for Lambda to use for the function. To create an execution role in the account and have Lambda use that one instead, choose Create, and then follow the on-screen instructions.
-lambda.function.enableXRay.tooltip=If selected, Lambda enables AWS X-Ray to detect, analyze, and optimize performance issues with the function.
-lambda.function.sourceBucket.tooltip=Choose an available Amazon S3 bucket in the connected AWS account for the AWS Serverless Application Model Command Line Interface (AWS SAM CLI) to use to deploy the function to Lambda, or create a new one by choosing Create button.
+lambda.function.name.tooltip=The function's name. It accepts A-Z, a-z, 0-9, -, and _. It must be less than 64 characters.
```

Comment:
I'd say a clearer way to word this would be "Can only contain alphanumeric (A-Z, a-z, 0-9), underscore (_) and hyphen (-) characters and must not exceed 64 characters in length."